.github/workflows/docs.yml

```diff
@@ -41,7 +41,7 @@ jobs:
 
       - uses: errata-ai/vale-action@reviewdog
         with:
-          files: docs/source/*.md
+          files: docs/source/
```

Comment:
This causes the build to stall and never complete.

https://github.com/plone/volto/actions/runs/4263176990/jobs/7420446300

Please monitor builds to reduce review cycles and to get immediate feedback for what does and does not work.